skipper.yaml

```diff
@@ -15,6 +15,9 @@ volumes:
   - $HOME/.docker/:$HOME/.docker
   - $KUBECONFIG:$KUBECONFIG
 
+  # podman - sharing the podman.socket between the host and the skipper container.
+  - /run/user/${UID}/podman/podman.sock:/run/podman/podman.sock
```

Comment:
Shouldn't it be part of skipper internals? Just like we don't pass the docker socket explicitly.

Reply:
not sure, it's unsecured so need to think about it carefully
Let's start with it this way
please read this thread: https://coreos.slack.com/archives/CBBJY9GSX/p1629391006061200

Reply:
It's the same amount of insecure with docker. I think it should be aligned on the skipper level as the skipper wasn't created as a security tool but a developer-friendly one. That runs as privileged with a host network, user info, etc.
@eranco74 WDYT?

Reply:
@YuviGold I agree that for administrator users it's reasonable but unlike docker you have to enable the socket (disabled by default) and need to think about  solution for unprivileged users (docker runs as root but podman isn't).
For now let continue without changing skipper.

Reply:
I'm fine with both options.
Let's start with this